app/controllers/AlertController.java

```diff
@@ -89,7 +90,7 @@ public Result addOrUpdate() {
         }
 
         try {
-            _alertRepository.addOrUpdateAlert(alert);
+            _alertRepository.addOrUpdateAlert(alert, Organization.DEFAULT);
```

Comment:
Because I didn't plumb the organization through all the APIs, as mentioned in the original comment on the PR.